doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -7,220 +7,125 @@ NumPy for MATLAB users
 Introduction
 ============
 
-MATLAB® and NumPy/SciPy have a lot in common. But there are many
-differences. NumPy and SciPy were created to do numerical and scientific
-computing in the most natural way with Python, not to be MATLAB clones.
-This page is intended to be a place to collect wisdom about the
-differences, mostly for the purpose of helping proficient MATLAB users
-become proficient NumPy and SciPy users.
+MATLAB® and NumPy have a lot in common, but NumPy was created to work with
+Python, not to be a MATLAB clone.  This guide will help MATLAB users get started
+with NumPy. 
 
 .. raw:: html
 
    <style>
    table.docutils td { border: solid 1px #ccc; }
    </style>
 
-Some Key Differences
+Some key differences
 ====================
 
 .. list-table::
 
-   * - In MATLAB, the basic data type is a multidimensional array of
-       double precision floating point numbers.  Most expressions take such
-       arrays and return such arrays.  Operations on the 2-D instances of
-       these arrays are designed to act more or less like matrix operations
-       in linear algebra.
-     - In NumPy the basic type is a multidimensional ``array``.  Operations
-       on these arrays in all dimensionalities including 2D are element-wise
-       operations.  One needs to use specific functions for linear algebra
-       (though for matrix multiplication, one can use the ``@`` operator
-       in python 3.5 and above).
-
-   * - MATLAB uses 1 (one) based indexing. The initial element of a
-       sequence is found using a(1).
+   * - In MATLAB, the basic type, even for scalars, is a
+       multidimensional array. Array assignments in MATLAB are stored as
+       2D arrays of double precision floating point numbers, unless you
+       specify the number of dimensions and type.  Operations on the 2D
+       instances of these arrays are modeled on matrix operations in
+       linear algebra. 
+
+     - In NumPy, the basic type is a multidimensional ``array``.  Array
+       assignments in NumPy are usually stored as ND arrays with the
```

Comment:
```suggestion
       assignments in NumPy are usually stored as n-dimensional arrays with the
```
`ND`, while consistent, might be cryptic. I think it's reasonable to say `n-dimensional`. I apologize if I steered you in the wrong direction.